### ISSUE TITLE
[Event Request] Codeunit 7041 "Price Asset - Item" - FillBestLine()

### ISSUE THREAD

**Issue by kmistryebecs** [microsoft/ALAppExtensions#6852]
I would like an event added to the FillBestLine() function in Codeunit 7041 "Price Asset - Item", this is needed when there are no Sales Prices and the price from the Item Card is used.  I would need the same parameters as the function itself, i.e....

`OnFillBestLine(PriceCalculationBuffer: Record "Price Calculation Buffer"; AmountType: Enum "Price Amount Type"; var PriceListLine: Record "Price List Line")
`

Basically I need to implement something similar to "Allow Line Disc." (for my own field) which would normally be set from a field on the Sales Price but when there is no Sales Price it needs to be set to the original value from the Sales Line.  In order to achieve this I need the above event.

**Comment by bc-ghost** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. We will update this issue with information about availability.
